clang/test/Driver/flang/reassoc.f90

```diff
@@ -0,0 +1,19 @@
+! REQUIRES: classic_flang
+
+! Tests for flags which generate nsw, reassoc attributes
+
+! RUN: %flang -Kieee %s -### 2>&1 | FileCheck %s
+! RUN: %flang -Kieee -fno-signed-zeros %s -### 2>&1 | FileCheck %s
+! RUN: %flang -Kieee -fassociative-math %s -### 2>&1 | FileCheck %s
+! RUN: %flang -Kieee -fassociative-math -fno-signed-zeros %s -### 2>&1 | FileCheck %s
+! RUN: %flang -Knoieee %s -### 2>&1 | FileCheck %s
+! RUN: %flang -Knoieee -fassociative-math %s -### 2>&1 | FileCheck %s
```

Comment:
Add coverage for other options that turn off `NeedIEEE`, e.g. `-ffast-math`, `-frelaxed-math`.

Reply:
added